sysid-application/src/main/native/cpp/view/Analyzer.cpp

```diff
@@ -29,6 +29,8 @@
 
 using namespace sysid;
 
+static double badDataGain = -9999.0;
```

Comment:
```suggestion
static constexpr double kBadDataGain = -9999.0;
```

Reply:
The compiler doesn't like the fact that I try to convert the const double* to a regular double* for the InputDouble method. Is there any way to work around that?

Reply:
Oh, ok.